packages/cli/src/commands/init/__tests__/validate.test.ts

```diff
@@ -28,6 +27,8 @@ test.each([
     name: 'helloworld_test',
     error: HelloWorldError,
   },
-])("'%s' is invalid name", ({name, error}: {name: string, error: Error}) => {
+  // @ts-ignore-next-line FIXME extending the Error class causes weird TS validation errors
+  // https://stackoverflow.com/questions/41102060/typescript-extending-error-class
+])("'%s' is invalid name", ({name, error}: {name: string; error: Error}) => {
```

Comment:
This is the only unsolved case from the conversion.
Apparently TS has weird issues when extending the `Error` class by not recognizing the correct properties.
Tried different solutions with no success. 😞